config/deploy.rb

```diff
@@ -0,0 +1,99 @@
+# config valid only for Capistrano 3.1
+lock '3.2.1'
+
+set :application, 'Reservations'
+set :repo_url, 'https://github.com/YaleSTC/reservations.git'
+
+# Default branch is :master
+set :branch, "#{ENV['GIT_TAG']}"
+
+# Default deploy_to directory is /var/www/my_app
+set :deploy_to, "#{ENV['DEPLOY_DIR']}"
+
+# Set Rails environment
+# set, :rails_env, 'production'
```

Comment:
I believe it's the default, we could probably just take it out. Note that this PR probably doesn't have a finalized Capistrano setup since I have no way to test it, it's really just for the isolation of configuration parameters in environment variables and the setup of `secrets.yml` and `.env` + documentation.